service.ts

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import crypto from 'node:crypto';
```

Comment:
You can import directly
```suggestion
import { randomUUID } from 'crypto';
```

Reply:
fixed